mlflow/java/src/test/java/org/mlflow/PandasDataFrameTest.java

```diff
@@ -0,0 +1,85 @@
+package org.mlflow.sagemaker;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.File;
+import org.mlflow.utils.SerializationUtils;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import org.mlflow.MLflowRootResourceProvider;
+
+import ml.combust.mleap.runtime.frame.DefaultLeapFrame;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+public class PandasDataFrameTest {
+  @Test
+  public void testPandasDataFrameIsProducedFromValidJsonSuccessfully() throws IOException {
+    String sampleInputPath =
+        MLflowRootResourceProvider.getResourcePath("mleap_model/sample_input.json");
+    String sampleInputJson = new String(Files.readAllBytes(Paths.get(sampleInputPath)));
+    PandasRecordOrientedDataFrame pandasFrame =
+        PandasRecordOrientedDataFrame.fromJson(sampleInputJson);
+    Assert.assertEquals((pandasFrame.size() > 0), true);
```

Comment:
Why not == expected size?

Reply:
Done! (Changed to `pandasFrame.size() == 1`)